### ISSUE TITLE
Add an option to combine search and create

### ISSUE THREAD

**Issue by fi0** [drgrib/alfred-bear#12]
What 're your thoughts on adding a combined 'search and create' option similar to that on nvALT or Notational Velocity?

For instance, when I search for "example".
it shows

```
note 1 example
note 2 example
create a new note titled 'example'
```

I think this should be an extra option or command, but not the default.

**Comment by drgrib** [comment]
I think it's a good idea. I frequently encounter the situation where I wanted to open a note if it exists or create it if it didn't many times. We could put it under `bsc` / `bsearchcreate`.

**Comment by jordaniusrex** [comment]
I wouldn't mind having it as a default single command. Show it as the last result when an exact match exists, and when the search doesn't find any results, it's the only option.

**Comment by drgrib** [comment]
Implemented in #15
